From f40aa3f74e05614425b094e465ea61db75cbfac4 Mon Sep 17 00:00:00 2001
From: KantaTamura <tkanta496@gmail.com>
Date: Mon, 10 Nov 2025 05:32:02 +0000
Subject: [PATCH 4/5] configs: enable mb command in dualboot defconfigs

Add CONFIG_CMD_MB=y to both xilinx_versal_dualboot_golden_defconfig and
xilinx_versal_dualboot_main_defconfig to include the non-secure 'mb'
command that directly manipulates the PMC_MULTI_BOOT register.

Signed-off-by: KantaTamura <tkanta496@gmail.com>
---
 configs/xilinx_versal_dualboot_golden_defconfig | 1 +
 configs/xilinx_versal_dualboot_main_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/configs/xilinx_versal_dualboot_golden_defconfig b/configs/xilinx_versal_dualboot_golden_defconfig
index 5298a30d5f8..489b43375c1 100644
--- a/configs/xilinx_versal_dualboot_golden_defconfig
+++ b/configs/xilinx_versal_dualboot_golden_defconfig
@@ -164,3 +164,4 @@ CONFIG_TPM=y
 CONFIG_DISTRO_BOOT_PARTITION_HELPERS=y
 CONFIG_BOOT_SEARCH_LABEL="bootg"
 CONFIG_BOOT_SCRIPT_NAME="boot.scr-golden"
+CONFIG_CMD_MB=y
diff --git a/configs/xilinx_versal_dualboot_main_defconfig b/configs/xilinx_versal_dualboot_main_defconfig
index 84e0ef71f46..ec276d0e11e 100644
--- a/configs/xilinx_versal_dualboot_main_defconfig
+++ b/configs/xilinx_versal_dualboot_main_defconfig
@@ -164,3 +164,4 @@ CONFIG_TPM=y
 CONFIG_DISTRO_BOOT_PARTITION_HELPERS=y
 CONFIG_BOOT_SEARCH_LABEL="bootm"
 CONFIG_BOOT_SCRIPT_NAME="boot.scr-main"
+CONFIG_CMD_MB=y
-- 
2.51.2

